Support/Testing/Travis/keychain.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
I think this script is fine for now, but putting keys in the repo is not the proper way of doing this. We can fix it later.